src/modules/launcher/Plugins/Microsoft.Plugin.Shell/ShellPluginSettings.cs

```diff
@@ -6,19 +6,19 @@
 
 namespace Microsoft.Plugin.Shell
 {
-    public class Settings
+    public class ShellPluginSettings
     {
-        public Shell Shell { get; set; } = Shell.RunCommand;
+        public ExecutionShell Shell { get; set; } = ExecutionShell.RunCommand;
 
         // not overriding Win+R
         // crutkas we need to earn the right for Win+R override
-        public bool ReplaceWinR { get; set; } = false;
+        public bool ReplaceWinR { get; set; }
```

Comment:
Was the false initialisation removed because we are planning to override Win+R?

Reply:
@alekhyareddy28 .  Nope this was to fix CA1805.  It is redundant to initialize a field to its default value.  

[Rule description](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1805?view=vs-2019#rule-description)
The .NET runtime initializes all fields of reference types to their default values before running the constructor. In most cases, explicitly initializing a field to its default value in a constructor is redundant, adding maintenance costs and potentially degrading performance (such as with increased assembly size), and the explicit initialization can be removed.